src/cache/cache.ts

```diff
@@ -1,28 +1,58 @@
-"use strict";
 
-import Redis from "ioredis";
+import { EitherAsync, Right, Left } from "purify-ts";
+import { Failure } from "../error/FailureType";
+import { PromiseError, NoDataRetrievedError } from "../error/ErrorFunctions";
+import { eitherPromiseToEitherAsync } from "../utils/EitherUtils";
+import { Cookie as string } from "../session/model/Cookie";
+import { SessionHandlerConfig } from "../SessionHandlerConfig";
+import { Redis } from 'ioredis';
+import IORedis = require('ioredis');
 
-import config from "../config";
+export class Cache {
```

Comment:
I think 'Cache' is a bit misleading here as we are using Redis for session storage and not as a cache

Reply:
Also the class name has an uppercase 'C' yet the filename is lowercase, we should be consistent here